src/h5web/toolbar/controls/DomainSlider/Histogram.module.css

```diff
@@ -16,3 +16,8 @@
 .bar {
   fill: var(--h5w-domainSlider-histogram--color, royalblue);
 }
+
+.indicators {
+  stroke: var(--h5w-domainSlider-histogram-indicator--color, midnightblue);
+  fill: var(--h5w-domainSlider-histogram-indicator--color, midnightblue);
```

Comment:
You can set the custom property on `color` and then use `currentColor` for both `fill` and `stroke`